docs/buildtools/dockerlogin/help.go

```diff
@@ -0,0 +1,12 @@
+package dockerlogin
+
+var Usage = []string{"docker login [command options]"}
+
+func GetDescription() string {
+	return `Login to a artifactory Docker registry`
+}
+
+func GetArguments() string {
+	return `	Command accepts optional registry for login. If not provided, the registry from the jfrog config will be used.
+	This argument is mandatory when logging in using username and password.`
```

Comment:
this section can also have arguments.
In case there are any arguments add it here.